src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/BookingEntity.kt

```diff
@@ -30,7 +30,7 @@ data class BookingEntity(
   val id: UUID,
   var crn: String,
   val arrivalDate: LocalDate,
-  val departureDate: LocalDate,
+  var departureDate: LocalDate,
```

Comment:
What is the significance of `val` vs `var`? Is it similar to `let` vs `const` in JS?

Reply:
Yeah same concept as `let` vs `const` - `var` can be reassigned later whereas `val` cannot